Modules/_ctypes/callproc.c

```diff
@@ -1123,6 +1123,15 @@ PyObject *_ctypes_callproc(PPROC pProc,
                 goto cleanup; /* leaking ? */
             }
         }
+#if (defined(__x86_64__) && (defined(__MINGW64__) || defined(__CYGWIN__))) || \
+        defined(__aarch64__)
+        void *tmp;
+        if (pa->ffi_type->type == FFI_TYPE_STRUCT) {
```

Comment:
Oh, I didn't see that issue--interesting.  I might be able to adapt the existing fix there to this issue as well.

As to whether or not it impacts just structs or any value > 8 bytes I'm not sure off the top of my head anymore.  I'll have to double-check.